src/main/java/com/toptrumps/core/DeckParser.java

```diff
@@ -1,4 +1,67 @@
 package com.toptrumps.core;
 
+
+import java.io.*;
+import java.util.ArrayList;
+
+
+// TODO: Remove this class
 public class DeckParser {
+
+    private ArrayList<Card> cards;
+
+    public DeckParser() {
+        buildDeck(); // calls the build deck method
+    }
+
+    private void buildDeck() {
+        cards = new ArrayList<>(); // creates a new array list
```

Comment:
Again, I don't think we need these kinds of comments anymore. Perhaps we should focus on explaining what each method does through a javadoc notation.